ParabolInc/parabol
# fix: copy invite link moves on scroll

# Description

Fixes [#7104]
I fixed the card dropdown menus to ensure their positions are fixed to their parent cards as required

## Demo
[loom video](https://www.loom.com/share/e799d2ad062744a9ab514c364cbb53b5)

## Testing scenarios

- [ ] On the meetings page
  - Click the menu button on any of the meeting cards
  - Scroll up or down
  - The menu should not be fixed to the same screen position, rather, it should move along with its parent card

## Final checklist
